gunicorn.conf.py

```diff
@@ -1,38 +0,0 @@
-# -*- coding: utf-8 -*-
-import os
-import socket
-
-if 'GUNICORN_TIMEOUT' in os.environ:
-    timeout = int(os.environ['GUNICORN_TIMEOUT'])
-
-# Smart detect heroku stack and assume a trusted proxy.
-# This is a convenience that should hopefully not be too surprising.
-if 'heroku' in os.environ.get('LD_LIBRARY_PATH', ''):
-    forwarded_allow_ips = '*'
-
-if 'H_GUNICORN_CERTFILE' in os.environ:
-    certfile = os.environ['H_GUNICORN_CERTFILE']
-
-if 'H_GUNICORN_KEYFILE' in os.environ:
-    keyfile = os.environ['H_GUNICORN_KEYFILE']
-
-def post_fork(server, worker):
-    # Support back-ported SSL changes on Debian / Ubuntu
-    import _ssl
-    import gevent.hub
-    if hasattr(_ssl, 'SSLContext') and not hasattr(_ssl, '_sslwrap'):
-        gevent.hub.PYGTE279 = True
```

Comment:
This looks like old Python 2 stuff and also Ubuntu specific. On my machine it doesn't get executed (the `if` evaluates to `False`) and I'm guessing it doesn't execute on prod either. I removed it